src/hotspot/share/ci/ciReplay.cpp

```diff
@@ -762,11 +763,26 @@ class CompileReplay : public StackObj {
         if (had_error()) {
           break;
         }
+        int inline_late = parse_int("inline_late");
+        if (had_error()) {
+          if (_version < 2) {
+            // Old format, ignore error and continue
+            _error_message = NULL;
+            inline_late = 0;
+            if (had_error()) {
+              // Pending exception?
+              break;
+            }
```

Comment:
I don't see how a pending exception is possible here, given the check at L763, and parse_int() doesn't throw any.
What do you think about not calling parse_int() if _version < 2, that way there is no error to ignore?

Reply:
I think that's a better idea to not parse it at all now that we have version numbers available.